python/GafferSceneUI/LightTweaksUI.py

```diff
@@ -137,6 +137,10 @@ def __init__( self, childPlug ) :
 		self.__row.append( GafferUI.PlugValueWidget.create( childPlug["mode"] ) )
 		self.__row.append( GafferUI.PlugValueWidget.create( childPlug["value"] ), expand = True )
 
+		button = GafferUI.Button( image = "delete.png", hasFrame = False )
+		self._deleteSignal = button.clickedSignal().connect( functools.partial( Gaffer.WeakMethod( self._removeTweak ), childPlug ) )
```

Comment:
This should be private (double underscore) rather than protected (single underscore). It's also not a signal - `self.__deleteConnection` would be more accurate.